### ISSUE TITLE
Delete script from popup

### ISSUE THREAD

**Issue by rayman89** [Tampermonkey/tampermonkey#547]
Could you add to the addon popup UI a way to delete the userscripts?

Something like this

![imagen](https://user-images.githubusercontent.com/5449042/40566482-75591456-6036-11e8-8333-961e33a5406e.png)

**Comment by rayman89** [comment]
yes? no? maybe?

**Comment by rayman89** [comment]
yes? no? maybe? @derjanb @grant

**Comment by derjanb** [comment]
I don't like this feature, because IMO the user has to confirm the action first and this is difficult to be done at a popup. As work-around just middle-mouse-click the script and delete it via editor menu: "File" > "Remove".

**Comment by wadawada** [comment]
It is true that confirming actions at popup can be difficult
I propose adding a delete button which adds a script to the delete queue and disables it
users can confirm and delete scripts on the delete queue

The reason to add a queue is that you can multi delete scripts activated on a web page
Right click on a script and remove it works for 1 script, but time consuming to open many script pages to delete them

**Comment by derjanb** [comment]
This issue should be fixed at TM BETA 4.12.6124. 😅

I'm not sure the Chrome Webstore team will be able to review this version this year, but in the meantime, you can download it from [here](https://www.tampermonkey.net/crx/tampermonkey_beta_4_12_6124.crx).

**Comment by wadawada** [comment]
> This issue should be fixed at TM BETA 4.12.6124. 😅
> 
> I'm not sure the Chrome Webstore team will be able to review this version this year, but in the meantime, you can download it from [here](https://www.tampermonkey.net/crx/tampermonkey_beta_4_12_6124.crx).

thanks for the update, but the crx link returns
>Not Found
>The requested URL was not found on this server.

**Comment by derjanb** [comment]
Oh, then please use the [most current version](https://www.tampermonkey.net/crx/tampermonkey_beta_4_12_6126.crx).

**Comment by wadawada** [comment]
I'm testing it.
Something is wrong when I try to import backups from stable to beta so I cannot really use it.
(imports loads indefinitely and never finish importing)
But I can still install scripts in beta to test the delete function.

It works as expected, except I think it can be improved by adding "undo delete".
Since confirming actions at popup can be difficult, there is no need to confirm the delete action at popup menu, instead, users will confirm actions by refreshing the page

In short, shows undo button after delete button is clicked
delete button will create delete flag, which can be undo by undo button before page is refreshed
if there is no undo action before refreshing the page, the script will keep delete state and will be deleted
